src/outfit.ts

```diff
@@ -205,6 +214,23 @@ export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?
       forceEquip.push($item`Kramco Sausage-o-Matic™`);
     }
   }
+
+  if (embezzlerUp) {
+    const currentWeapon = 25 * findLeprechaunMultiplier(meatFamiliar());
+    const value = ((75 - currentWeapon) * (750 + baseMeat)) / 100;
+    const embezzlers = globalOptions.ascending
+      ? 30
+      : Math.min(30, embezzlerCount() || digitizedMonstersRemaining());
+    if (value * embezzlers > mallPrice($item`scratch 'n' sniff UPC sticker`)) {
+      acquire(3, $item`scratch 'n' sniff UPC sticker`, value * 30);
```

Comment:
30 seems like a magic number given that stickers fall off after 20 turns (I assume you're using a buffer of 1.5x?). I would think `value * embezzlers` is a good hard cap, since that's the exact value at which stickers become unprofitable after.

Also, do we need to always acquire 3? I guess there's no major harm since we won't purchase any more if we already have >3 in our inventory (and used none in previous `useUPCs()`), but we could also in theory acquire only as many stickers as we don't already have stuck on our sword.